prov/tcp/src/tcpx_attr.c

```diff
@@ -67,8 +73,9 @@ static struct fi_domain_attr tcpx_domain_attr = {
 	.control_progress = FI_PROGRESS_AUTO,
 	.data_progress = FI_PROGRESS_AUTO,
 	.resource_mgmt = FI_RM_ENABLED,
+	.mr_mode = OFI_MR_BASIC_MAP | FI_MR_SCALABLE | FI_MR_BASIC,
```

Comment:
Does the provider actually support basic registration?

Reply:
I am using the util code ofi_mr_reg* functions for memory registration. I see that key is assigned in those functions. Does it mean tcp provider support FI_MR_BASIC?

Reply:
Yes, I checked code, utility MR supports FI_MR_BASIC

Reply:
Got it -- thanks.  But the provider doesn't require OFI_MR_BASIC_MAP mr_mode bits.  That is, it can support FI_MR_BASIC for backwards compatibility, but shouldn't set other mr_mode bits as needed.

Reply:
> I see that key is assigned in those functions.

Right, it allocates from RB tree keys if provider has `FI_MR_PROV_KEY` capability

Reply:
> but shouldn't set other mr_mode bits as needed.

Thanks, OFI_MR_BASIC_MAP mr_mode bits were removed